library/src/morpheus-master/sass/modules/_toolmenu.scss

```diff
@@ -500,6 +500,8 @@ nav#subSites{
 			&.is-current{
 				position: relative;
 				margin-right: -1px;
+				font-size: 1.2em;
```

Comment:
I am concerned by the font-size making the menu bounce when changing tools. If you want to increase the font-size of the selected tool, then you'll need to allocate the minimum space on the tool so the item height doesn't change. This cannot be done using height though, because some long tool names may wrap. Line height might be a useful property to use to allocate the space.